src/layman/error_list.py

```diff
@@ -7,7 +7,7 @@
     6: (500, 'Cannot connect to database'),
```

Comment:
In DB methods, use combination (schema, table_name) instead of (workspace, table_name), if reasonable